app/src/main/java/io/github/droidkaigi/confsched/util/LocaleUtil.java

```diff
@@ -103,6 +103,20 @@ public static Date getDisplayDate(@NonNull Date date, Context context) {
         }
     }
 
+    public static Date getConfTimezoneCurrentDate() {
```

Comment:
Sorry, now I'm home.

I wanted to say like below. setTimeZone would be called at the first time LocaleUtil being loaded.
And you can cache DateFormat instances on the memory:D

```java
    private static final DateFormat DATE_FORMAT_TOKYO = SimpleDateFormat.getDateTimeInstance();
    private static final DateFormat DATE_FORMAT_LOCAL = SimpleDateFormat.getDateTimeInstance();
    static {
           DATE_FORMAT_TOKYO.setTimeZone(CONFERENCE_TIMEZONE);
           DATE_FORMAT_LOCAL.setTimeZone(TimeZone.getDefault());
    }
    public static Date getConfTimezoneCurrentDate() {
        Date date = new Date();
        try {
            return DATE_FORMAT_LOCAL.parse(DATE_FORMAT_TOKYO.format(date));
        } catch (ParseException e) {
            Log.e(TAG, "date: " + date + "can not parse." + e);
            return date;
        }
    }
```